bitnami/external-dns/README.md

```diff
@@ -188,6 +188,7 @@ The following table lists the configurable parameters of the external-dns chart
 | `podSecurityContext.runAsUser`         | User ID for the container                                                                                                                                                                                       | `1001`                                                  |
 | `priorityClassName`                    | priorityClassName                                                                                                                                                                                               | `""`                                                    |
 | `securityContext`                      | Security context for the container                                                                                                                                                                              | `{}`                                                    |
+| `service.create`                       | Whether to create Service resource or not                                                                                                                                                                       | `true`                                                  |
```

Comment:
In the Consul chart we're using `service.enabled`, could you adapt to that (for consistency purposes)?

Reply:
fixed